### ISSUE TITLE
Incorrect preview size

### ISSUE THREAD

**Issue by cenosito** [tdlib/telegram-bot-api#442]
When I upload videos with my bot, it displays the size of the video distorted. However, if I upload the video by dragging it directly from my computer, the image and size of the video appear in the chat without distortion.
The md5sum of mp4 file uploaded is equal in both files.

Right size:
https://postimg.cc/xJj6k3dM

Wrong size:
https://postimg.cc/1grW5jyy

I have also noticed that when uploading it through the bot, the video duration is not displayed, but if I upload it manually, it does.

In short, the video duration is not shown when uploading it through the bot, and the thumbnail, which should be 180x320, is displayed as 320x320.

**Comment by levlam** [comment]
This is expected if the bot doesn't specify correct parameters in the [sendVideo](https://core.telegram.org/bots/api#sendvideo) request.

**Comment by cenosito** [comment]
> This is expected if the bot doesn't specify correct parameters in the [sendVideo](https://core.telegram.org/bots/api#sendvideo) request.

Thank you for your prompt and accurate response! We can consider the matter closed.

I thought that since the server was set to optional, it would automatically provide these parameters like when you upload a video manually, but I see that each parameter needs to be configured manually.

**Comment by cenosito** [comment]
Solved
